_maps/map_files/Theseus/TGS_Theseus.dmm

```diff
@@ -12936,7 +12939,7 @@
 	dir = 1
 	},
 /turf/open/floor/mainship/emerald{
-	dir = 5
+	dir = 10
```

Comment:
Why are you varediting turfs all over the place?

Reply:
Minor tile fixes, since I went over each map to make sure the changes I made didn't mess up the existing floors.